### PR TITLE
add VS Code IDE DuckDB related tools to SQL Clients section (#21)

### PR DESCRIPTION
**Pull request by RandomFractals**
as described in https://github.com/davidgasquez/awesome-duckdb/issues/21.

Updated SQL Clients and IDE tools section in our fork for this PR to preview updated doc:

https://github.com/RandomFractals/awesome-duckdb#sql-clients-and-ide-that-support-duckdb

We've included our Premium Pro data tools extensions as well, as they provide direct integration with DuckDB SQL Tools in VS Code IDE for PRQL docs and various markdown documents some DuckDB users might be interested to explore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,8 +45,12 @@
 
 ## SQL Clients and IDE that Support DuckDB
 
+- [DuckDB SQL Tools](https://marketplace.visualstudio.com/items?itemName=RandomFractalsInc.duckdb-sql-tools) - Free DuckDB SQL Tools for VS Code IDE.
+- [DuckDB Pro Tools](https://github.com/RandomFractals/pro-data-tools/blob/main/duckdb-tools.md#duckdb-pro-tools) - Premium DuckDB Tools for VS Code IDE with advanced DuckDB connection features and extended database schema views.
 - [Harlequin](https://harlequin.sh) - The DuckDB IDE for your terminal. ([GitHub](https://github.com/tconbeer/harlequin)).
 - [qStudio](https://www.timestored.com/qstudio/) - A free SQL tool specialized for data analysts. It runs on every operating system and allows easy browsing of tables and charting of results.
+- [PRQL Pro Tools](https://github.com/RandomFractals/pro-data-tools/tree/main#prql-pro-tools) - Premium PRQL Tools for VS Code IDE with PRQL Run Query Code Lens and DuckDB SQL Tools support.
+- [Markdown SQL Pro Tools](https://github.com/RandomFractals/pro-data-tools/tree/main#markdown-sql-pro-tools) - Premium Markdown SQL Tools for VS Code IDE with SQL Code Lenses and DuckDB SQL Tools integration.
```
Comment (davidgasquez) [inline README.md:53]:
```suggestion
- [Harlequin](https://harlequin.sh) - The DuckDB IDE for your terminal. ([GitHub](https://github.com/tconbeer/harlequin)).
- [qStudio](https://www.timestored.com/qstudio/) - A free SQL tool specialized for data analysts. It runs on every operating system and allows easy browsing of tables and charting of results.
- [DuckDB SQL Tools](https://marketplace.visualstudio.com/items?itemName=RandomFractalsInc.duckdb-sql-tools) - Free DuckDB SQL Tools for VS Code IDE. [Premium version available](https://github.com/RandomFractals/pro-data-tools/blob/main/duckdb-tools.md#duckdb-pro-tools) with advanced features.
```

**Review by davidgasquez — CHANGES_REQUESTED** [review]
Heya @RandomFractals! Thanks for the PR and the VSCode SQL Tools!

I've sent a suggestion covering a couple of things:

- Sort by addition date. I haven't figured out the best way to sort each section. Until I figure it out or discover the official way to do it, I've decided to keep it simple.
- Coalesce tools into one line. Feels like SQL Tools and SQL Pro Tools are similar enough to be in the same project bullet point. I've removed the other two as they're more related with PRQL/Markdown than DuckDB. If you still think they should be here, let's add them under one line at the `Project Powered by DuckDB`.

Thanks again!

**Comment by RandomFractals** [comment]
@davidgasquez That's fine for now. DuckDB users using VS Code could benefit from PRQL Code Lens and new Markdown SQL Tools extension that are totally different tools and hook into PRQL and markdown document editors to run DuckDB SQL queries with VS Code SQL Tools.

Also, DuckDB Pro Tools is a separate extension, not just a premium version of the free DuckDB SQL Tools extension with advanced features, but we are sure users can discover them all from our updated tools docs that cover them more in detail with demos and examples.

Let's merge with your changes for now. Thanks!

**Comment by davidgasquez** [comment]
Awesome! Thanks for the extra details!

**Pull request merged**